packages/protocol/contracts/dapis/Api3Adminship.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+import "./interfaces/IApi3Adminship.sol";
+
+/// @title Adminship contract controlled by the API3 DAO
+contract Api3Adminship is IApi3Adminship {
+  string internal constant ERROR_UNAUTHORIZED = "Unauthorized";
+  string internal constant ERROR_ZERO_ADDRESS = "Zero address";
+  string internal constant ERROR_EXPIRATION_NOT_EXTENDED = "Expiration not extended";
+
+  /// @dev Meta admin sets the admin statuses of addresses and has super
+  /// admin privileges
+  address public metaAdmin;
+  mapping(address => AdminStatus) public adminStatuses;
+
+  /// @dev Reverts if the caller is not the meta admin or has admin/superadmin priviliges
+  modifier onlyAdminOrMetaAdmin() {
+    require(adminStatuses[msg.sender] >= AdminStatus.Admin || msg.sender == metaAdmin, ERROR_UNAUTHORIZED);
```

Comment:
I think this may be redundant? Since `SuperAdmin` is in the `AdminStatus` struct.